{"version":1,"ops":[{"type":1,"author":{"id":"6d59fc378929d99368e023bcfc49aacae1e68aad"},"timestamp":1246050879,"metadata":{"github-id":"MDU6SXNzdWUyODQ4NTEw","github-url":"https://github.com/parrot/parrot/issues/313","origin":"github"},"title":"Bug in subroutine calling conventions","message":"When running this code with the command 'parrot -':\n\n```\n.sub 'main' :main\n    'aux'(5, 'hello', \"b\" =\u003e 3.14)\n.end\n\n.sub 'aux' \n    .param int a\n\n    .param num b :named(\"b\")\n\n    .param string c :optional\n\n    .param int has_c :opt_flag\n\n    say a\n\n    say b\n\n    say c\n.end\n```\n\nI get this error:\n\n```\nFixedIntegerArray: index out of bounds!\n\ncurrent instr.: 'aux' pc 16 (-:5)\n\ncalled from Sub 'main' pc 11 (-:2)\n```\n\nOriginally http://trac.parrot.org/parrot/ticket/792 by Axle","files":null}]}